_drafts/2017-10-13-rprofile-david-smith.md

```diff
@@ -0,0 +1,118 @@
+---
+name: rprofile-david-smith
+layout: post_discourse
+title: ".rprofile: David Smith"
+date: 2017-10-13
+authors:
+  - name: Kelly O'Briant
+    url: https://kellobri.github.io/
+tags:
+- R
+- community
+- interviews
+---
+
+<div class="alert alert-info" role="alert">
+In this occasional series, we interview someone using a loosely defined set of interview questions for the purpose of “Demystifying the Creative/Development Process of the R Community”. This interview was conducted and prepared by Kelly O'Briant as part of an rOpenSci unconf17 project.
```

Comment:
@sckott Can you please replace "...for the purpose of “Demystifying the Creative/Development Process of the R Community”. 

with 
...for the purpose of demystifying the creative and development processes of R community members.

Reply:
will do